openstudiocore/src/utilities/core/StringHelpers.cpp

```diff
@@ -273,4 +274,17 @@ std::string formatUnderscore(const std::string& str) {
   return wStr;
 }
 
+std::vector <std::string> splitString(const std::string &string, char delimiter)

```

Comment:
Looks good to me, my only genius idea is we might want to add arguments to keep empty parts or not, maybe an overload that splits on a string too:

http://qt-project.org/doc/qt-4.8/qstring.html#split